cmd/gardener-scheduler/app/gardener_scheduler.go

```diff
@@ -111,175 +78,104 @@ func NewCommandStartGardenerScheduler() *cobra.Command {
 			if err := opts.validate(args); err != nil {
 				return err
 			}
-			return opts.run(cmd.Context())
+
+			return runCommand(cmd.Context(), opts)
 		},
 	}
 
 	flags := cmd.Flags()
 	verflag.AddFlags(flags)
 	opts.AddFlags(flags)
+
 	return cmd
 }
 
-// GardenerScheduler represents all the parameters required to start the
-// Gardener scheduler.
-type GardenerScheduler struct {
-	Config            *config.SchedulerConfiguration
-	Identity          *gardencorev1beta1.Gardener
-	GardenerNamespace string
-	ClientMap         clientmap.ClientMap
-	Logger            *logrus.Logger
-	Recorder          record.EventRecorder
-	LeaderElection    *leaderelection.LeaderElectionConfig
-}
+func runCommand(ctx context.Context, opts *Options) error {
+	// Load config file
+	config, err := configloader.LoadFromFile(opts.ConfigFile)
```

Comment:
Variable `config` collides with imported package name 
```suggestion
	cfg, err := configloader.LoadFromFile(opts.ConfigFile)
```